installer/PowerToysSetupCustomActions/CustomAction.cpp

```diff
@@ -227,16 +227,19 @@ UINT __stdcall CreateScheduledTaskCA(MSIHANDLE hInstall) {
 
   // ------------------------------------------------------
   //  Save the task in the PowerToys folder.
-  hr = pTaskFolder->RegisterTaskDefinition(
-    _bstr_t(wstrTaskName.c_str()),
-    pTask,
-    TASK_CREATE_OR_UPDATE,
-    _variant_t(username_domain),
-    _variant_t(),
-    TASK_LOGON_INTERACTIVE_TOKEN,
-    _variant_t(L""),
-    &pRegisteredTask);
-  ExitOnFailure(hr, "Error saving the Task : %x", hr);
+  {
```

Comment:
A few line above:
```
hr = pPrincipal->put_RunLevel(TASK_RUNLEVEL_HIGHEST);
```

needs to be changed to 

```
hr = pPrincipal->put_RunLevel(TASK_RUNLEVEL_LUA);
```

Reply:
Actually it didn't work for me, after installing the MSI on a clean machine, it still restarts as an admin. I'll do more testing to try to figure out what is happening on my machine,